source/less/blocks/page-header.less

```
@@ -0,0 +1,32 @@
+.page-header {
+  background-color: @granite;
+}
+
+.page-header__toggle {
+  width: 23px;
+  height: 23px;
+  background-color: transparent;
+  border: none;
+  position: relative;
+  cursor: pointer;
+
+  &::after {
+    content: "";
+    width: 100%;
+    height: 2px;
+    position: absolute;
+    background-color: @white;
+    left: 0;
+    transform: rotate(45deg);
+  }
+
+  &::before {
+    content: "";
+    width: 100%;
+    height: 2px;
+    position: absolute;
+    background-color: @white;
+    left: 0;
+    transform: rotate(-45deg);
+  }
```

Можно избавиться от дублирования

```suggestion
  &::before,
  &::after {
    content: "";
    width: 100%;
    height: 2px;
    position: absolute;
    background-color: @white;
    left: 0;
    transform: rotate(45deg);
  }

  &::before {
    transform: rotate(-45deg);
  }
```